src/modules/core/components/Tag/Tag.tsx

```diff
@@ -6,7 +6,9 @@ import { useMainClasses } from '~utils/hooks';
 import styles from './Tag.css';
 
 interface Appearance {
-  theme: 'primary' | 'light' | 'golden' | 'danger';
+  theme: 'primary' | 'light' | 'golden' | 'danger' | 'pink' | 'blue';
+  fontSize?: 'tiny';
+  colorSchema?: 'inverted' | 'plain';
```

Comment:
we should have a default color schema, for the "full color" version of the tag:

![Screenshot from 2021-03-12 11-28-51](https://user-images.githubusercontent.com/1193222/110920516-29e80700-8326-11eb-8bd2-90d2fb3dbea7.png)

You can name it however you'd like, just that it needs to be there so that we can explicitly set it _(right now, the only way to "set it" it to leave the `colorSchema` prop empty)_